js/ui/interaction.js

```diff
@@ -36,6 +36,7 @@ function Interaction(el) {
     scrollwheel(zoom);
     el.addEventListener('dblclick', ondoubleclick, false);
     window.addEventListener('resize', resize, false);
+    window.addEventListener('keydown', keydown, false);
```

Comment:
My question is more why does it bind keydown on document when focused and then unbind when blurred, rather than binding once to the map container?